docs/personalization/integrating_results.md

```diff
@@ -0,0 +1,36 @@
+# Integrating scenario results
+
+When the personalization service is [enabled](enabling_personalization.md) and properly 
+[configured](perso_configuration.md), as an editor, you can embed the recommendations 
+that come from the service, to show them to the end users.
+You can, for example, modify the landing page to include a block that shows what 
+content items/products are recommended to end users when specific [scenarios](scenarios.md) are triggered. 
+Depending on the scenario type, you may need to provide additional information 
+to see the recommendation results.
+
+The number and selection of available scenarios depends on the arrangements that 
+your organization makes with Ibexa when defining the initial configuration.
+
+1. Navigate to the **Site** > **List** tab, and then click the **Preview** 
+icon next to a site, in which you want to place a personalization block.
+
+1. On the Page Builder screen, click the **Preview/Edit** selector to switch to editing mode.
+
+1. From the **Elements** menu, drag and drop the **Personalized** block to a location on the page layout.
```

Comment:
Side note: the block should also be listed in https://doc.ibexa.co/projects/userguide/en/master/site_organization/working_with_page/#block-settings